metaflow/plugins/kubernetes/kubernetes_decorator.py

```diff
@@ -79,6 +78,7 @@ class KubernetesDecorator(StepDecorator):
         "namespace": None,
         "gpu": None,  # value of 0 implies that the scheduled node should not have GPUs
         "gpu_vendor": None,
+        "tolerations": None,  # e.g., [{"key": "arch", "operator": "Equal", "value": "amd"}]
```

Comment:
Two questions:

- The map could also have a key called `effect`. So should the comment be changed to something like `# e.g., [{"key": "arch", "operator": "Equal", "value": "amd", "effect":"NoSchedule"}]`. e.g. [this](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
- There's an additional comment above for allowing multiple tolerations. In that case, would users express the tolerations as something like:
`[{"key": "arch", "operator": "Equal", "value": "amd"}],[{"key": "arch", "operator": "Equal", "value": "arm"}]`